src/formatters/html.js

```diff
@@ -20,6 +20,10 @@ function htmlEscape(text) {
   return html;
 }
 
+HtmlFormatter.prototype.prepareContext = function(context) {
+  BaseFormatter.prototype.prepareContext.call(this, context);
+};
```

Comment:
The fix solved the problem, also annotated worked because of the call to super. Sorry, but it would take me ages to reproduce with the exact setup we have. Thanks for your awesome library!